tests/test_config_manager.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (C) 2015  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from tests.support import mock, RepoStub
+
+import config_manager
+import dnf
+import filecmp
+import iniparse
+import os
+import shutil
+import tempfile
+import unittest
+
+
+REPOLABEL = "testrepo"
+REPOCONTENT = """[testrepo]
+name=TestRepo
+baseurl=file:///tmp
+enabled=1
+"""
+
+class ConfigManagerBase(mock.MagicMock):
+      conf = dnf.conf.Conf()
+
+class ConfigManagerCommandTest(unittest.TestCase):
+
+    def setUp(self):
+        cli = dnf.cli.cli.Cli(ConfigManagerBase())
+        self.cmd = config_manager.ConfigManagerCommand(cli)
+        self.cmd.base.conf.reposdir = [tempfile.mkdtemp()]
+
+        self.addCleanup(shutil.rmtree, self.cmd.base.conf.reposdir[0])
```

Comment:
No, ```addCleanup``` is better because it is run even if ```setUp``` fails. In this case, it does not matter. In other cases, ```adCleanup``` is better.